servers/rendering/shader_language.h

```diff
@@ -966,6 +968,7 @@ class ShaderLanguage {
 	int completion_line;
 	BlockNode *completion_block;
 	DataType completion_base;
+	bool completion_base_array;
```

Comment:
Should probably have a default initialization?

Reply:
Added it to `clear()` function.